snapshot/src/main/java/io/zeebe/snapshots/broker/ConstructableSnapshotStore.java

```diff
@@ -12,7 +12,7 @@
 import java.util.Optional;
 
 /** A persisted snapshot store than can create a new snapshot and persists it. */
```

Comment:
Would be cool if we can adjust the docs then

Reply:
Isn't the doc already up-to-date?

Reply:
Hm ok maybe you're right. I thought maybe using this word 'constructing' in the doc would fit here as well. But feel free to reject this comment

Reply:
Ah.. ok. Let me see if I can improve it. Thanks.